notes/decentralization-autonomy-organisation-brekke.md

```diff
@@ -0,0 +1,76 @@
+- 1. viloent market expansion: "systematically destroying all other means for people to collectively resolve issues and access the things that they need "
+- 2. structural stupididy: " if you can see with your own eyes that people are starving needs are not being met, there is climate disasters happening ,homelessness or whatever else there's a kind of ideological story here for why you're not supposed to intervene in the market
+- decentralization in time: direct democracy, market dictatorship, market design
+- market design : hold market accoutable
+- "in a similar way tokens and and token driven governance or token based governance you know will suit some people and not others"
+- "i think there's **a lot** more critical thinking that needs to be go into it is that complete merge of economic dynamics and governance that we see in the idea of radical markets"
+- "i do think that we need to keep an eye on what does it mean to approach the world through the lens of economics" 
+- "keep an eye on network effects and scale" "scale and difference can oftentimes clash" tysons anti scale argument
+- 90s don't hat ethe media - be the media, indi media channels - a chance to epress local content and context?
+- decentralization in space: on what layer
+- spacce, air, environment, community, user , value, application, network, governance, protocol, virtual machine, energy , land
+- "there was another unfolding that also happened which was the awareness of whales(monopolies) the consciousness of whales in the space"
+- moxie ref when APIs run only throuh companies
+- it is more important to think about what we want to achieve strategicaly than to adhere to a abstract technical concept of "decentralization"
+- "Decentralization as a strategy not ideology"
+- "like moxie said admiting that not everyone is going to run his own server/node"
+- "I just want to open up these concepts so that they are considered less as ideologies less as kind of absolute goods and more as as as strategies as as pragmatics to try and achieve something so that we can actually start to be a little bit more clear on the things that we're actually really trying to achieve"
+- "I like to come back and end up with the policy report that i wrote for for the european commission because it was really some a kind of process where i was thinking .. whether the arts might be a kind of gateway for people who are involved in crypto to actually start experimenting more with some actually functioning and upgraded forms of democracy for the digital age"  
+- euopean comission report: https://publications.jrc.ec.europa.eu/repository/handle/JRC121675 : d
+SCANNING THE EUROPEAN ECOSYSTEM OF DISTRIBUTED LEDGER FOR SOCIAL AND PUBLIC GOOD TECHNOLOGIES - What, Why, Where, How, and Ways to Move Forward - Alexandre Pólvora (editor), Anna Hakami (editor),Erica Bol (editor) ,Samer Hassan,
```

Comment:
Should this and next few lines be one line?

Reply:
Hi @rufuspollock I changed all the things you mentioned here on my main branch so it should be fine now 👍